matchms/filtering/add_retention.py

```diff
@@ -10,7 +10,8 @@
 logger = logging.getLogger("matchms")
 
 
-_retention_time_keys = ["retention_time", "retentiontime", "rt", "scan_start_time", "RT_Query"]
+_retention_time_keys = ["retention_time", "retentiontime", "rt", "scan_start_time",
+                        "RT_Query", "rt_query"]

```

Comment:
That's right. I wanted to first discuss the Metadata harmonization part, but I would indeed argue to keep all keys lower case within matchms so that the "RT_Query" could go.

Reply:
I removed the upper case version.